packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -42,15 +42,19 @@
     user-select: none;
     width: var(--terra-consumer-nav-width, 320px);
     z-index: 2;
-
     @media screen and (max-width: $terra-tiny-breakpoint) {
       width: 100vw;
     }
 
     @media screen and (max-width: $terra-medium-breakpoint) {
+      height: calc(100% - 60px) !important;
```

Comment:
why important and where is 60px coming from?

Reply:
If the 60px is the profile height, i'm confused why line 39 is not fixing this problem as that is in place to address this issue.

Reply:
can this just be applied on line 35? Also shouldn't need important